docs/removing-republish-button-from-categories.md

```diff
@@ -0,0 +1,25 @@
+# Removing the Republish Button From Specific Categories
+
+If you'd like to remove the Republish button from specific categories, the plugin includes a filter that will help you do this programatically. 
+
+The `hide_republication_widget` filter allows you to target specific posts/categories/tags/whatever you want to hide. All you have to do is check if the current post exists in the specific category, and tell the filter to return `true` if you'd like the button to be hidden.
+
+Here's an example of how to hide the Republish button on all posts in the category with an ID of 14.
+
+```
+function remove_republish_button_from_category(){
```

Comment:
This example function needs to accept 1 parameter, and return the value of that parameter if the conditional is not set.

It should also have a docblock.

Reply:
Docblock added in https://github.com/INN/republication-tracker-tool/pull/54/commits/17b1480cc99d5d5bd6c5626b6a2293261606d57a.

@benlk Can't we just `return false;` if the conditional isn't met?

Reply:
We cannot. There may be other filters here running at other priorities that set modify the value, or the post metabox may be set to `true`. Filters should not assume that the default value is set or unset; all that we can reasonably expect as the author of this filtering function is that the documentation regarding the type of the expected parameters is correct.

Reply:
@benlk The doc function is updated in https://github.com/INN/republication-tracker-tool/pull/54/commits/45e4fd516691c883758fc70f3a70800235079f11.

Reply:
@benlk https://github.com/INN/republication-tracker-tool/pull/54/commits/45e4fd516691c883758fc70f3a70800235079f11 updates the sample function to include the new params and returns the value of `$hide_republication_widget`.

Reply:
👍